app/models/scoreboard.rb

```diff
@@ -1,7 +1,10 @@
-class ScoreboardSetup < ActiveRecord::Base
```

Comment:
This is more in line with the Autograder, and it makes more sense from a modeling perspective.  An Assessment has a Scoreboard or doesn't, and if it has a Scoreboard, then that scoreboard's settings will be in the Scoreboard model.  Plus, this is shorter, which is a plus in my book.